internal/store/migrate.go

```diff
@@ -0,0 +1,28 @@
+package store
+
+import (
+	"errors"
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+	"strings"
+)
+
+func MigrateDb(dbURI string) error {
```

Comment:
nit: Acronyms should be upper cased: `MigrateDB()`
https://github.com/golang/go/wiki/CodeReviewComments#initialisms